src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -76,7 +76,16 @@ static class ParseException extends Exception {
         }
     }
 
-    private enum Phase { PREAMBLE, BODY, POSTAMBLE }
+    private enum Phase {
+        /** The initial part of an HTML file up to and including the {@code body} and possible {@code <main>} tag. */
+        PREAMBLE,
+        /** The initial part of a doc comment, or the rich-text content of a block tag. */
```

Comment:
Curious: is "initial part of a doc comment" the same as what we now call [main description](https://docs.oracle.com/en/java/javase/20/docs/specs/javadoc/doc-comment-spec.html#main-description)?